documentation/src/main/markdown/downloadingandrunning/ephemeralscan.md

```diff
@@ -0,0 +1,112 @@
+    - name: component3
+      version: version3
+```
+
+Each policy override must apply to a list of specific components, on a specific version (e.g. component1 + version1) or on all versions (e.g. component2).
+
+## Results
+
+Unlike persistent scans, no data is stored on Black Duck and all scans are done transiently. These scans are primarily intended to be fast, although the SIGNATURE_SCAN can take some time as communication with Black Duck is a requirement.
+
+The results are saved to a json file named 'name_version_BlackDuck_DeveloperMode_Result.json' in the Scan Output directory, where name and version are the project's name and version.
+
+The results are also printed in the logs:
+```
+2021-07-20 13:25:18 EDT INFO  [main] --- Ephemeral Scan Result: (for more detail look in the log for Ephemeral Scan Result Details)
+2021-07-20 13:25:18 EDT INFO  [main] ---
+2021-07-20 13:25:18 EDT INFO  [main] --- 		Critical and blocking policy violations for
+2021-07-20 13:25:18 EDT INFO  [main] --- 			* Components: 0
+2021-07-20 13:25:18 EDT INFO  [main] --- 			* Security: 5
+2021-07-20 13:25:18 EDT INFO  [main] --- 			* License: 0
+2021-07-20 13:25:18 EDT INFO  [main] ---
+2021-07-20 13:25:18 EDT INFO  [main] --- 		Other policy violations
+2021-07-20 13:25:18 EDT INFO  [main] --- 			* Components: 101
+2021-07-20 13:25:18 EDT INFO  [main] --- 			* Security: 0
+2021-07-20 13:25:18 EDT INFO  [main] --- 			* License: 0
+2021-07-20 13:25:18 EDT INFO  [main] ---
+2021-07-20 13:25:18 EDT INFO  [main] --- 		Policies Violated:
+2021-07-20 13:25:18 EDT INFO  [main] --- 			Security Vulnerabilities Great Than Or Equal to High
+2021-07-20 13:25:18 EDT INFO  [main] --- 			Warn on Low Security Vulnerabilities
+2021-07-20 13:25:18 EDT INFO  [main] --- 			Warn on Medium Security Vulnerabilities
+2021-07-20 13:25:18 EDT INFO  [main] ---
+2021-07-20 13:25:18 EDT INFO  [main] --- 		Components with Policy Violations:
+2021-07-20 13:25:18 EDT INFO  [main] --- 			Apache PDFBox 2.0.12 (maven:org.apache.pdfbox:pdfbox:2.0.12)
+2021-07-20 13:25:18 EDT INFO  [main] --- 			Handlebars.js 4.0.11 (npmjs:handlebars/4.0.11)
+2021-07-20 13:25:18 EDT INFO  [main] ---
+2021-07-20 13:25:18 EDT INFO  [main] --- 		Components with Policy Violation Warnings:
+2021-07-20 13:25:18 EDT INFO  [main] --- 			Acorn 5.5.3 (npmjs:acorn/5.5.3)
+```
+
+## Ephemeral Scan Compare Mode
+
+You can configure Ephemeral scan to return only the difference in policy violations between previous rapid scans. To return only the difference in policy violations, configure detect.blackduck.rapid.compare.mode to BOM_COMPARE or BOM_COMPARE_STRICT.
+
+ALL evaluates all RAPID/EPHEMERAL or FULL policies. BOM_COMPARE_STRICT only shows policy violations not present in an existing project version BOM. BOM_COMPARE depends on the type of
+policy rule modes and behaves like ALL if the policy rule is only RAPID but like BOM_COMPARE_STRICT when the policy rule is RAPID and FULL. See the Black Duck documentation for
```

Comment:
"BOM_COMPARE depends on the type of policy rule modes and behaves like ALL if the policy rule is only RAPID but like BOM_COMPARE_STRICT when the policy rule is RAPID and FULL. See the Black Duck documentation for complete details."

Perhaps a little change like:

BOM_COMPARE depends on the type of policy rule modes selected and behaves like ALL if the policy rule is only RAPID and like BOM_COMPARE_STRICT when the policy rule is RAPID and FULL. See the Black Duck documentation for complete details.

Reply:
Yours is better